text_localization_environment/TextLocEnv.py

```diff
@@ -1,65 +1,82 @@
 import gym
-from gym import spaces
 import random
+import numpy as np
+from gym import spaces
 from gym.utils import seeding
 from chainer.backends import cuda
 from PIL import Image, ImageDraw
 from PIL.Image import LANCZOS, MAX_IMAGE_PIXELS
-import numpy as np
-from text_localization_environment.ImageMasker import ImageMasker
+from .ImageMasker import ImageMasker
+from .transformer import create_bbox_transformer
+from .utils import scale_bboxes
 
 
 class TextLocEnv(gym.Env):
     metadata = {'render.modes': ['human', 'rgb_array', 'box']}
 
-    DURATION_PENALTY = 0.03
+    # Length of history in state
     HISTORY_LENGTH = 10
```

Comment:
The history length needs also to be configured for the agents model, so I don't think this is the best place to put it.
It should be a parameter to the environment.